docs/jwst/tweakreg/astrometric_utils.rst

```diff
@@ -0,0 +1,15 @@
+=================
+astrometric_utils
+=================
+
+The ``astrometric_utils`` module provides functions for generating
+astrometric catalogs of sources for the field-of-view covered by a
+set of images.
+
+.. moduleauthor:: Warren Hack <help@stsci.edu>
```

Comment:
@nden and I were just discussing the issue of having author names in the code and/or docs and came to the conclusion that this is perhaps not the best thing to do, since responsibility for modules changes so frequently, and hence would be better to just give credit via a list of contributors for the whole package. Also, is the email address "help@stsci.edu" even active anymore?

Reply:
All such attributions found in this step's docs were removed in efc699a.